analyzer/java/com/engflow/bazel/invocation/analyzer/suggestionproviders/BottleneckSuggestionProvider.java

```diff
@@ -129,6 +131,21 @@ public SuggestionOutput getSuggestions(DataManager dataManager) {
               .map(bottleneckStats -> generateSuggestion(bottleneckStats, totalDuration))
               .collect(Collectors.toList());
 
+      final List<Caveat> caveats = new ArrayList<>();
+      var flagForTargetInclusionEnabled =
+          dataManager.getDatum(FlagValueExperimentalProfileIncludeTargetLabel.class);
+      if (suggestions.size() > 0
+          && !flagForTargetInclusionEnabled.isProfileIncludeTargetLabelEnabled()) {
+        caveats.add(
+            SuggestionProviderUtil.createCaveat(
+                String.format(
+                    "The profile does not include which target each action was processed for,"
+                        + " although that data can help with breaking down bottlenecks. It is added"
+                        + " to the profile by using the Bazel flag `%s`. Also see %s",
+                    FlagValueExperimentalProfileIncludeTargetLabel.FLAG_NAME,
+                    FlagValueExperimentalProfileIncludeTargetLabel.COMMAND_LINE_REFERENCE_URL),
+                false));
+      }
```

Comment:
```suggestion
      var flagForTargetInclusionEnabled =
          dataManager.getDatum(FlagValueExperimentalProfileIncludeTargetLabel.class);
      var caveats = !suggestions.isEmpty()
          && !flagForTargetInclusionEnabled.isProfileIncludeTargetLabelEnabled()
          ? ImmutableList.of(SuggestionProviderUtil.createCaveat(
                String.format(
                    "The profile does not include which target each action was processed for,"
                        + " although that data can help with breaking down bottlenecks. It is added"
                        + " to the profile by using the Bazel flag `%s`. Also see %s",
                    FlagValueExperimentalProfileIncludeTargetLabel.FLAG_NAME,
                    FlagValueExperimentalProfileIncludeTargetLabel.COMMAND_LINE_REFERENCE_URL),
                false))
          ? ImmutableList.of();
```

Reply:
The code below potentially adds another caveat, so the suggested change doesn't work.

Reply:
See line 171

Reply:
https://github.com/EngFlow/bazel_invocation_analyzer/pull/122/files#diff-7156908f0e818c7d4dd0bea6eaee9dcc8ad505517cd3ceeed355400773f3bde2R171